docstore-metrics/src/main/java/org/hypertrace/core/serviceframework/docstore/metrics/DocStoreMetricsRegistry.java

```diff
@@ -167,11 +167,14 @@ public StandardDocStoreMetricsRegistry() {
     }
 
     private void monitor() {
-      executor.scheduleAtFixedRate(
-          this::queryDocStoreAndSetMetricValues,
-          INITIAL_DELAY_SECONDS,
-          standardMetricsReportingInterval.toSeconds(),
-          SECONDS);
+      // TODO: Disabled for fixing ENG-57034
```

Comment:
Please delete entirely, don't check in commented out code - especially given that the comment indicates there's no plan to restore it.